test/studies/adventOfCode/2022/day11/bradc/day11-par.chpl

```diff
@@ -0,0 +1,181 @@
+use IO, List, Barriers;
+
+config const numRounds = 20;
+
+// things that define a monkey
+class Monkey {
+  var id: int,                 // an integer ID
+      Items: [0..1] list(int, parSafe=true),  // two lists of its items
```

Comment:
Why is `Items` specifically uppercase?

Reply:
Unless I've missed any, I've been capitalizing all of my major array's names in this series.  That said, this one doesn't meet my normal "this is a major array" reasons for capitalizing them, so I probably shouldn't have.